spec/models/card_spec.rb

```diff
@@ -1,26 +1,27 @@
 require "rails_helper"
 
 describe Card do
-  test_card = Card.create(original_text: "Тест",
-                          translated_text: "test ")
+  before :each do
+    @test_card = FactoryGirl.create :card
```

Comment:
Оффтопом с сайта ruby-doc.org ))

let(name, &block)
Essentially, define an accessor for name with block.
Why use let instead of def? I honestly don’t know.